src/Sylius/Bundle/ResourceBundle/Controller/RequestConfiguration.php

```diff
@@ -470,7 +470,7 @@ public function getEvent()
     public function hasPermission()
     {
         if (!$this->parameters->has('permission')) {
-            return true;
+            return false;
         }
 
         return false !== $this->parameters->get('permission');
```

Comment:
`return true === $this->parameters->get('permission');` would be safer approach (e.g. `false !== 'false'` - will return true...).